src/arrays.js

```diff
@@ -1,61 +1,83 @@
 const getNthElement = (index, array) => {
-  // your code here
+  index %= array.length;
+  return array[index];
```

Comment:
Eslint doesnt like this because you are reassigning the index parameter inside the function, you should never do this as it can lead to unexpected behaviour in larger functions. You can use basically the exact same logic you have here in a single line and not do this reassignment:
```
  return array[index % array.length];
```